datamade/just-spaces
# Add basic CSV export

## Overview

Closes #80. Implements basic DataTables CSV export button and improves DataTables styling. Intended to look like this:

<img width="921" alt="Screen Shot 2019-06-03 at 4 30 28 PM" src="https://user-images.githubusercontent.com/1094243/58835939-f2aad900-861c-11e9-9029-89b0d8833dc2.png">

## Testing Instructions

 * Look at a survey with submitted data
* Can you view that data in the table on the page?
* Click the "Download as CSV" button. Does it look like you'd expect?
